src/generator/src/cmd/generate.ts

```diff
@@ -248,35 +259,55 @@ async function findReadmePaths(specsPath: string) {
   });
 }
 
-async function buildTypeIndex(logger: ILogger, baseDir: string, apiVersion?: ApiVersion) {
-  const extensionBaseDir = apiVersion == undefined ? generatedOutDir : path.join(baseDir, apiVersion, extensionConfig[apiVersion].version);
+async function buildTypeIndex(logger: ILogger, baseDir: string, apiVersion: ApiVersion) {
+  // Add the MsGraphBicepExtensionConfig type to the last position in types.json file
+  function addConfigToContent(content: string): any[] {
+    const contentTypes = JSON.parse(content) as any[];
+    const relationshipType = contentTypes.find(type => type["$type"] === TypeBaseKind.ObjectType && type["name"] === 'MsgraphRelationship');
```

Comment:
Not sure if this is where the problem is, but the relationship type should be called MicrosoftGraphRelationship.
We should not be abbreviating Microsoft to MS in any of our type names.